test/tools/test_case.py

```diff
@@ -32,14 +33,16 @@ class TestCase:
     config = None
     time_started = time()
     timeout = None
+    timeout_of_entire_test = None
 
-    def __init__(self, name, config, timeout=80000):
+    def __init__(self, name, config, timeout=10, timeout_of_entire_test=80000):
```

Comment:
remove "timeout" input parameter because it is not used

Reply:
ok

Reply:
No. It's used. See
```
    def __init__(self, name, config, timeout=10, timeout_of_entire_test=80000):
        self.name = name
        self.deployer = Deployer(config)
        self.agent = Agent(config)
        self.blockchain = BlockChain(config)
        self.config = config
        self.timeout = timeout ### <-------- here <-------- <-------- <--------
        self.timeout_of_entire_test = timeout_of_entire_test

```
Also, having `timeout` in constructor is standard in these py tests.

Reply:
So why do you implement a new variable `timeout_of_entire_test` if we can use the old one - as it was before the changes

Reply:
`timeout_of_entire_test` is max time to run 1 test and wait it's complete
`timeout` is 1 operation timeout inside test
due to some reason these 2 timeouts were just 1 timeout previously
they are different

Reply:
I see, timeout is passing to the agent functions - and a timeout of some operation

And you just added a timout_of_entire_test